README.md

```diff
@@ -149,6 +149,17 @@ Grape::Idempotency.configure do |c|
 end
 ```
 
+### processing_response
```

Comment:
Naming to be reviewed

Reply:
What do you think about including this explanation under the `How it works` section? This last section is for configuration.